demos/habib-2017/src/convert_to_zarr.py

```diff
@@ -0,0 +1,53 @@
+import argparse
+from anndata import read_h5ad
+from vitessce.data_utils import (
+    to_uint8,
+    sort_var_axis,
+    optimize_adata,
+)
+
+
+def convert_h5ad_to_zarr(input_path, output_path):
+    adata = read_h5ad(input_path)
+
+    # Store an expression matrix with only the highly variable genes.
+    adata = adata[:, adata.var['highly_variable']].copy()
```

Comment:
why?

Reply:
to be more specific, i don't remember what is in this variable - is it a few thousand genes and we show them all?  is just a few hundred, a small subset of thousands?  if it's the later, it seems like we could just use "slicing" + "filter" to load it, which is generally recommended (by me).  even so, we could just make `highly_variable` a filter.